dhis-2/pom.xml

```diff
@@ -35,6 +35,7 @@
     <module>dhis-support</module>
     <module>dhis-web-api</module>
     <module>dhis-web-embedded-jetty</module>
+    <module>dhis-support/dhis-support-test-json</module>
   </modules>
```

Comment:
There is also some significance to the _order_ of modules. We want the module to be put as early as possible in the module build order. Modules which depend on other modules must come after those modules, and so there must be an implicit hierarchy to avoid circular dependencies. This particular module has no dependencies on other DHIS 2 modules and so it can be placed first I think.

Reply:
Good point.